llvm/lib/Transforms/InstCombine/InstCombineLoadStoreAlloca.cpp

```diff
@@ -270,16 +270,54 @@ class PointerReplacer {
     unsigned ToAS = ASC->getDestAddressSpace();
     return (FromAS == ToAS) || IC.isValidAddrSpaceCast(FromAS, ToAS);
   }
+  bool FoundASC(const Value *TrueOp) const;
```

Comment:
```suggestion
  bool hasFoundASC(const Value *TrueOp) const;
```

Based on the LLVM code standard, function name starts with a verb and non-capitalized letter.

Reply:
And why is it called `TrueOp`?

Reply:
A leftover name from refactoring into a function. Thanks for catching it.